src/Infrastructure/Serialization/Symfony/Normalizer/EnumNormalizer.php

```
@@ -6,18 +6,27 @@
     {
         return [\BackedEnum::class => true];
     }
 
+    /** @return bool True when $data is a backed enum instance */
     public function supportsNormalization(mixed $data, ?string $format = null, array $context = []): bool
     {
         return $data instanceof \BackedEnum;
     }
 
+    /** @return string The scalar backing value of the enum case */
     public function normalize(mixed $object, ?string $format = null, array $context = []): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$format'. (undefined)

(UnusedFormalParameter)